.github/workflows/update_ci_resources.yml

```diff
@@ -0,0 +1,26 @@
+name: update ci resources
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *' # 1 AM every day
+
+jobs:
+  test_matrix:
+    strategy:
+      matrix:
+        platform:
+          - name: K3s-QEMU-SNP
+            runner: SNP
+          - name: K3s-QEMU-TDX
+            runner: TDX
+          - name: K3s-QEMU-SNP-GPU
+            runner: SNP-GPU
+    name: "${{ matrix.platform.name }}"
+    runs-on: ${{ matrix.platform.runner }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Update kubernetes resources
+        run: |
+          kubectl apply -f ./tools/bm-maintenance/deployment_tdx_snp.yml
```

Comment:
The image referenced in the bm-maintenance job is untagged, and I have no idea where it comes from - how about building and pushing it in a predecessor step?

Reply:
Still untagged, and we also push this elsewhere at the same time.